images/cgimap/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ruby:3.3.0 AS builder
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV workdir=/var/www
+ENV CGIMAP_GITSHA=8ea707e10aeab5698e6859856111816d75354592
+ENV cgimap=/tmp/openstreetmap-cgimap
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential cmake git curl file \
+    libxml2-dev libpqxx-dev libfcgi-dev zlib1g-dev libbrotli-dev \
+    libboost-program-options-dev libfmt-dev libmemcached-dev libcrypto++-dev \
+    libargon2-dev libyajl-dev \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/zerebubuth/openstreetmap-cgimap.git $cgimap \
+ && cd $cgimap \
+ && git checkout $CGIMAP_GITSHA \
+ && mkdir build && cd build && cmake .. && cmake --build .
```

Comment:
I don't see the additional build flags we're using in https://github.com/zerebubuth/openstreetmap-cgimap/blob/v2.0.1/docker/debian/Dockerfile_bookworm#L23

Production binaries should be compiled in "Release" mode...